src/__tests__/App.test.js

```diff
@@ -0,0 +1,16 @@
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import App from '../App';
+import '@testing-library/jest-dom/extend-expect';
+import store from '../redux/store';
+
+describe('app routes', () => {
+  it('app elements', () => {
+    render(
+      <Provider store={store}>
+        <App />
+      </Provider>,
+    );
+    expect(screen.getByTestId('home')).toBeInTheDocument();
+  });
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests%20for%20pure%20functions%20in%20your%20Redux%20code.), Please also create unit tests for pure functions in your Redux code.